go-controller/pkg/node/node.go

```diff
@@ -243,34 +259,52 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	go n.gateway.Run(n.stopChan, wg)
 	klog.Infof("Gateway and management port readiness took %v", time.Since(start))
 
+	isOvnUpEnabled, err := getOVNIfUpCheckMode()
+	if err != nil {
+		return err
+	}
+	// we use atomic lib to store port binding mode state, so use int32 to represent bool
+	var ovnPortBinding int32
+	if isOvnUpEnabled {
+		ovnPortBinding = 1
+	}
```

Comment:
@trozet can we keep the int stuff inside NewCNIServer() and just pass `isOvnUpEnabled` into it?

Reply:
ok